pkg/process/checks/process.go

```diff
@@ -101,6 +108,8 @@ func (p *ProcessCheck) Init(_ *SysProbeConfig, info *HostInfo) error {
 	p.maxBatchSize = getMaxBatchSize()
 	p.maxBatchBytes = getMaxBatchBytes()
 
+	p.skipAmount = uint32(ddconfig.Datadog.GetInt32("process_config.checks_between_hints"))
```

Comment:
Let's try coming up with a bit more descriptive name for this? This doesn't appear to have relation with `process_discovery`.

Reply:
Renamed